src/docs/PropsTable.tsx

```diff
@@ -32,7 +32,7 @@ export const PropsTable: React.FC<PropsTableProps> = ({ props: componentProps }:
                                 {prop.name}
                             </Text>
                         </TableCell>
-                        <TableCell>
+                        <TableCell style={{ whiteSpace: 'normal' }}>
```

Comment:
We wanted to break the line inside of the `PropsTable` inside the documentation as we have many different options for the `Placement` of a Tooltip, adding the whiteSpace with either `normal` or `break-spaces` seems to do the deal with the following:

![image](https://user-images.githubusercontent.com/9327569/184671331-dc53ceeb-9ed3-4c88-bd1c-ea796fbdf113.png)

Initially Leo and I thought about having a parameter to send to the PropsTable to define this wrapping, but I've checked all other tables and it does not seem to affect them as this might be the first one that we have so many possible options. So I'm adding this directly to the Type's `TableCell` if that's okay. If you have any other recommendation to achieve this, let me know please :)